### DISCUSSION TITLE
Modifying game saves

### DISCUSSION THREAD

**Discussion by Bandmo** [sanni/cartreader#404] [Need help]
Hey everyone,

So one of the things I was hoping to be able to do with my cart reader is hack game saves. I accidentally erased my son's pokemon blue game save, and I wanted to get him back his progress on pokemon without him completely restarting.

I have been trying to use PKHeX to try to alter the save file, but it doesn't seem to be the right format on file. Anyone done this before? I could use some help. Thanks!

**Comment by Dakkaron** [comment]
Pokemon save data isn't that complicated and there aren't really different formats, so I'd first try to check if the dump worked correctly.

What Pokemon game are we talking about here? The generations differ greatly, same as the systems they were run on.

Can you maybe upload your savegame somewhere?

What problems did you have with pkhex?

**Reply by Bandmo** [reply]
I figured it out! It was a bad dump. Dirty contacts. I salvaged my kid's childhood!!!... and maybe slipped him a huge collection of pokeballs and their variants :-D
